python/lsst/ip/diffim/modelPsfMatch.py

```diff
@@ -451,8 +451,8 @@ def _buildCellSet(self, exposure, referencePsfModel):
             paddingPix = self.config.padPsfBy
 
         if paddingPix > 0:
-            self.log.info("Padding Science PSF from (%s, %s) to (%s, %s) pixels" %
-                          (psfSize, psfSize, paddingPix + psfSize, paddingPix + psfSize))
+            self.log.debug("Padding Science PSF from (%d, %d) to (%d, %d) pixels",
+                           psfSize, psfSize, paddingPix + psfSize, paddingPix + psfSize)
```

Comment:
You're ok with losing this one to `debug`?

Reply:
Yes, I think that's a good level for diffim, also.